homeassistant/components/brunt/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for brunt integration."""
+import logging
+
+from aiohttp import ClientResponseError
+from aiohttp.client_exceptions import ServerDisconnectedError
+from brunt import BruntClientAsync
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class BruntConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for youless."""
+
+    VERSION = 1
+    _reauth_entry = None
+
+    def __init__(self):
+        """Start the Brunt config flow."""
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(step_id="user", data_schema=DATA_SCHEMA)
+
+        errors = None
+        bapi = BruntClientAsync(
+            username=user_input[CONF_USERNAME], password=user_input[CONF_PASSWORD]
+        )
+        try:
+            await bapi.async_login()
+        except ClientResponseError as exc:
+            if exc.status == 403:
+                errors = {"base": "invalid_auth"}
+            else:
+                _LOGGER.warning("Unknown error when connecting to Brunt: %s", exc)
```

Comment:
Unknown errors are logged including stack trace in the config flow.
```suggestion
                _LOGGER.exception("Unknown error when connecting to Brunt: %s", exc)
```